### DISCUSSION TITLE
WPF Local Versions

### DISCUSSION THREAD

**Discussion by jogibear9988** [dotnet/wpf#6038] [Q&A] [answered]
Hello,

some years ago (don't know exactly when), there was a feature planed to introduce WPF local versions (so I could deploy my app including WPF dll's in a specific version). I thought there was a Video about it.

Is this Video still available somewhere? And what happend to that feature?

**Comment by ThomasGoulet73** [comment]
Hey @jogibear9988,
Could you be referring to [self-contained deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained) ? This includes .Net itself (Including WPF in a WPF application) in the deployment which isolates the application and does not require .Net to be installed on the machine that runs the application.

**Comment by jogibear9988** [comment]
Maybe now... But I'm search for the Video where this was anounced for WPF years before NetCore. There were plans, so WPF could be updated, but does not break old Apps. Thought it was called WPF local Version.

**Comment by singhashish-wpf** [comment]
@jogibear9988 This has been converted to a discussion. We will also try to search for the video internally and share if available.

**Comment by Felix-Dev** [comment, marked as answer]
@jogibear9988 I *think* you could mean [this presentation](https://youtu.be/q40PSMWmoOI?t=136).

**Reply by jogibear9988** [reply]
That was not directly the one I searched for, but it keeps the message from that time. Thanks

**Reply by jogibear9988** [reply]
Great that you searched that for me :-)
